ocs_ci/ocs/perfresult.py

```diff
@@ -95,17 +95,16 @@ def dump_to_file(self):
 
     def es_write(self):
         """
-        Writing the results to the elastic-search server, if data can not be
-        written to the elasticsearch server, it will be written to JSON file
+        Writing the results to the elastic-search server, and to a JSON file
 
         """
 
         # Adding the results to the ES document / JSON file
         self.add_key("all_results", self.all_results)
-        log.info(json.dumps(self.results, indent=4))
+        log.debug(json.dumps(self.results, indent=4))
+        self.dump_to_file()
```

Comment:
can we add a log on the path of the JSON file for better readability?

Reply:
The path to the json file is logged in the dump_to_file function, on Line #92